### DISCUSSION TITLE
Some information about latest windows 11 beta update(drag-and-drop)

### DISCUSSION THREAD

**Discussion by QZLin** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#74] [General]
Microsoft seems to support drag-and-drop to the taskbar
In the recent beta channel update.

`Windows Feature Experience Pack 1000.22593.1.0`

[Announcing Windows 11 Insider Preview Build 22593](https://blogs.windows.com/windows-insider/2022/04/06/announcing-windows-11-insider-preview-build-22593/) (Not found in the changelog, but it works on my computer.)

Anyway, I would like to express my thanks to this project and to the authors. This tool is really useful and has helped me a lot.

**Comment by HerMajestyDrMona** [comment]
Yes, it was already for a while in the Preview Channels. I only wonder how fast it will be officially released as "stable" :) Hopefully faster than October 2022 as I was predicting.

I don't recommend switching to the BETA channel for people who only miss the native drag and drop. These test builds might be too unstable, that using this fix might still be a better solution.
